salt_stack/pillar/lambda.sls

```diff
@@ -0,0 +1,5 @@
+# Install from binary:
+node:
+  #version: 10.17.0
```

Comment:
This will be bumped to 10.x, soon.

Reply:
Yes. I tried compiling with 10.x but the dynamodb autoscale lambda fails. When we get the lambda updated then we need to update this as well.

Reply:
Updated to Node 10.x after pulling in update lambda code